yannickl88/features-bundle/1.0/etc/packages/features.yaml

```diff
@@ -0,0 +1,21 @@
+features:
+    tags:
+#        my.feature:
+#            request: ["beta", "on"]
+
+services:
```

Comment:
These services should be removed, as they should be added by the dev in the main `container.yaml` file.